tools/dialog/main.go

```diff
@@ -0,0 +1,36 @@
+package main
```

Comment:
Let's add a comment explaining that this tool can be used to test Zenity/Kdialog settings. (No need for README.md being so small a tool :)